services/management/ia/templates_service_test.go

```diff
@@ -50,17 +50,15 @@ func TestCollect(t *testing.T) {
 	sqlDB := testdb.Open(t, models.SkipFixtures, nil)
 	db := reform.NewDB(sqlDB, postgresql.Dialect, reform.NewPrintfLogger(t.Logf))
 
-	t.Run("bad and missing template paths", func(t *testing.T) {
+	t.Run("bad template paths", func(t *testing.T) {
 		t.Parallel()
 
 		testDir, err := ioutil.TempDir("", "")
 		require.NoError(t, err)
 		defer os.RemoveAll(testDir) //nolint:errcheck
```

Comment:
It's not used anywhere anymore. But the whole idea was to not create test data in default directories as it affects run `managed` in devcontainer.  When you run tests they create some template/rule files, and then `manage` loads these files.

Reply:
nice catch, I think it would be good to re-introduce it and put our converted rules in the tempDir, I will add it again in the next push thanks

Reply:
added the tempdir to `TestConvert` and removed it from `TestCollect` because we are not generating any rules there